peekingduck/cli.py

```diff
@@ -136,8 +136,21 @@ def init(custom_folder_name: str) -> None:
     help="""Modify node configs by wrapping desired configs in a JSON string.\n
          Example: --node_config '{"node_name": {"param_1": var_1}}'""",
 )
-def run(config_path: str, node_config: str) -> None:
+@click.option(
+    "--log_level",
+    default="info",
+    help=("""Modify log level {"error", "warning", "info", "debug"}"""),
+)
+def run(config_path: str, node_config: str, log_level: str) -> None:
     """Runs PeekingDuck"""
+    log_level_settings = set(["CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"])
+    log_level = log_level.upper()
+    if log_level not in log_level_settings:
+        logger.info(f"Unknown log_level {log_level}, defaulting to INFO")
+        log_level = "INFO"
+    if log_level != "INFO":
+        logger.info(f"Changing log_level to {log_level}")
```

Comment:
Possible to shift some of this into `LoggerSetup` as a `get_log_level` class method? So that both usage of `log_level_settings` can be ensured to be consistent.

Reply:
Abstracted into new static method `LoggerSetup.set_log_level()`.